_posts/2019-07-19-mock-parse-buildlog.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Mock can now parse the log from build
```

Comment:
I would go with something like "Parse user-friendly errors from Mock build logs" ... just an example, it can be something totally different. The point is that from the original title I know, that I can parse something from mock logs, but I don't know what and why should I care :-)